### PR TITLE
SafeBreach Code Review

### PR DESCRIPTION
**Pull request by gtsafeB**

### CODE REVIEW

File: Packs/SafeBreach/Integrations/SafeBreach v2/SafeBreach.py [hunk 1]
```diff
@@ -453,63 +454,64 @@ def get_insights_command(client: Client, args: dict, no_output_mode: bool) -> li
...
+            }
+            headers = list(t.keys())
+            insight_output.append(context_insight)
+            insight_readable.append(t)
+
+        readable_output = tableToMarkdown(name='SafeBreach Insights', t=insight_readable, headers=headers,
+                                          removeNull=True)
+
+        outputs = {
+            'SafeBreach.Insight(val.Id == obj.Id)': insight_output
```
Comment (esikuriansky) [inline Packs/SafeBreach/Integrations/SafeBreach v2/SafeBreach.py]: Is  'SafeBreach.Insight(val.Id == obj.Id)' the key or should if be evaluated by using f"" ?

Comment (gtsafeB) [reply Packs/SafeBreach/Integrations/SafeBreach v2/SafeBreach.py]: Its Demisto syntax not a string format

File: Packs/SafeBreach/Integrations/SafeBreach v2/README.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+e
+B
+r
+e
+a
+c
+h
+ 
+v
+2
```
[the remark this answers is not in the document]
Comment (gtsafeB) [reply Packs/SafeBreach/Integrations/SafeBreach v2/README.md]: Already fixed in newer commits

File: Packs/SafeBreach/Integrations/SafeBreach v2/SafeBreach.py [hunk 3]
```diff
@@ -0,0 +1,575 @@
...
+
+                standard_context_list.append(primary_standard_context)
+
+                dbot_score_list.append(dbot_score)
+
+            if len(standard_context_list) > 0:
+                standard_context_dict[demisto_standard_path] = standard_context_list
+                if secondary_path:
+                    standard_context_dict[secondary_path] = secondary_standard_context
+                    # if len(ip_list) > 0:
```
Comment (arossert) [inline Packs/SafeBreach/Integrations/SafeBreach v2/SafeBreach.py]: remove comments if not needed

**Pull request closed without merging**